src/main/resources/db/mysql/populateDB.sql

```diff
@@ -89,6 +89,10 @@ INSERT IGNORE INTO visits
 VALUES (3, 8, '2009-06-04', 'neutered');
 INSERT IGNORE INTO visits
 VALUES (4, 7, '2008-09-04', 'spayed');
+INSERT IGNORE INTO visits
```

Comment:
I would recommend updating the dates for these like in hsql, in case anybody uses those database engines